File: Scala_App/src/main/scala/StartApp.scala

import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.layout.AnchorPane
import javafx.scene.Scene
import javafx.stage.Stage



object StartApp {
  def main(args: Array[String]): Unit = Application.launch(classOf[StartApp], args: _*)

}
final class StartApp extends Application {

  var Stage: Stage = _
  var screenWidth: Double = _
  var screenHeight: Double = _

  override def start(stage: Stage): Unit = {
    Stage = stage
    val root: AnchorPane = FXMLLoader.load(getClass.getResource("menu.fxml"))
    val scene: Scene = new Scene(root, Config.screenWidth, Config.screenHeight)
    stage.setScene(scene)
    stage.show()
  }
}

File: Java_App/src/main/java/com/example/acceltone/StartApp.java
package com.example.acceltone;

import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Scene;
import javafx.scene.layout.AnchorPane;
import javafx.stage.Stage;

public class StartApp extends Application {

    private Stage stage;

    public static void main(String[] args) {
        launch(args);
    }

    @Override
    public void start(Stage primaryStage) throws Exception {
        this.stage = primaryStage;

        AnchorPane root = FXMLLoader.load(getClass().getResource("main.fxml"));
        Scene scene = new Scene(root, Config.screenWidth, Config.screenHeight);
        primaryStage.setScene(scene);
        primaryStage.show();
    }
}

